bin/generate.py

```diff
@@ -1261,18 +1275,30 @@ def parse(key, name, yaml, parent):
                 fatal(f'Could not parse {parent.path / name} as a testcase or directory.')
 
             if is_testcase(yaml):
+                count = 1
+                if yaml is not None and 'count' in yaml and isinstance(yaml['count'], int):
+                    count = yaml['count']
+
                 if not config.COMPILED_FILE_NAME_REGEX.fullmatch(name + '.in'):
                     error(f'Testcase \'{parent.path}/{name}.in\' has an invalid name.')
                     return None
 
-                # If a list of testcases was passed and this one is not in it, skip it.
-                if not self.process_testcase(parent.path / name):
-                    return None
+                ts = []
+                for i in range(count):
+                    # If a list of testcases was passed and this one is not in it, skip it.
+                    if not self.process_testcase(parent.path / name):
```

Comment:
Can you make it so that both `name` and `curname` trigger processing?

Reply:
i think only curname should trigger it?

Reply:
Hmm; `curname` should trigger for sure.
One case for `name` triggering it: if you increase the count, those are not files yet so you can't use globbing, and then writing `name` is simpler. But anyway this is rarely used to anything is fine.

Reply:
but thats also the case if you add new rules?

Reply:
ah good  point. `curname` only is good then.